app/controllers/discovered_hosts_controller.rb

```diff
@@ -173,9 +177,9 @@ def auto_provision_all
         end
       end
```

Comment:
Here is one drawback of this approach. We are no longer able to find rules and provision in one transaction. But since we want to be interactive, we have to do this...